index.js

```diff
@@ -91,40 +90,42 @@ function buildInternal(inputsAndConfig) {
     config.access = ['public', 'undefined', 'protected'];
   }
 
-  var parseFn = (config.polyglot) ? polyglot : parseJavaScript;
+  var parseFn = config.polyglot ? polyglot : parseJavaScript;
 
   var buildPipeline = pipeline(
     inferName,
     inferAccess(config.inferPrivate),
     inferAugments,
     inferKind,
+    nest,
     inferParams,
     inferProperties,
     inferReturn,
     inferMembership(),
     inferType,
-    nest,
```

Comment:
I don't believe so: nest nests tags (properties, params) - `hierarchy` nests according to membership. I'm excited to refactor hierarchy too as soon as this is release-ready. But there might be something I'm missing here that isn't uncovered by test fixtures?